lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -603,6 +603,19 @@ def update_ipv6_address(self, ue_id, ipv6_addr):
             ip6 = ipaddress.ip_address(ipv6_addr)
             self._ue_ip_map[ue_id] = ip6
 
+    def run_ipv6_data(self, ipv6_addr):
+        """Run ipv6 data"""
+        self.magma_utils = MagmadUtil(None)
+        install_scapy_cmd = "sudo pip3 install scapy"
+        execute_icmpv6_cmd = (
+            "sudo python3 /home/vagrant/magma/lte/gateway/python/icmpv6.py "
```

Comment:
I would suggest including this script (and also move it under `../scripts/`) into the `setup.py` on lte/gateway/python so it is installed as part of the python venv